### PR TITLE
main.go for SignUp

### PR DESCRIPTION
**Pull request by Krimyllex**
`main.go`: updated
`router.go`: modified

Closes: #159

### CODE REVIEW

File: cmd/ims/main.go [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+
+	// Open DB
+	db, err := sqlx.Connect("pgx", dsn)
+	if err != nil {
+		logrus.Fatalf("cannot connect to DB: %s", err.Error())
+	}
+
+	defer db.Close()
+
+	signupService := service.NewIMSService(db, nil, time.Time{})
```
Comment (taaraora) [inline cmd/ims/main.go:42]: what means that arg? `time.Time{}`

File: cmd/ims/config.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package main
+
+import "github.com/omc-college/management-system/pkg/db"
+
+type Config struct {
+	DBConnection 	db.ConnectionConfig 	`mapstructure:"dbconnection"`
+	WebAPIAddress 	string 			  		`mapstructure:"webapi_address"`
```
Comment (taaraora) [inline cmd/ims/config.go:7]: it has excessive spaces between tag and field type

File: cmd/ims/main.go (hunk 1 above)
Comment (Krimyllex) [reply cmd/ims/main.go:42]: It`s a nil value of type time.Time

File: cmd/ims/config.go (hunk 2 above)
Comment (Krimyllex) [reply cmd/ims/config.go:7]: It's a bug of GitHub. There is not such a big space

Comment (Krimyllex) [reply cmd/ims/config.go:7]: That's how it looks like on my desktop

File: cmd/ims/main.go (hunk 1 above)
Comment (Krimyllex) [reply cmd/ims/main.go:42]: I can`t just write `nil` here, it occurs an error, so I use a nil value of type time.Time

Comment (taaraora) [reply cmd/ims/main.go:42]: please make config property for it

**Pull request merged**